src/enclave-agent/src/services/images.rs

```diff
@@ -136,42 +100,45 @@ mod test {
 
     #[test]
     fn test_get_container_id() {
-        struct ParseCase{
+        struct ParseCase {
```

Comment:
let's move formatting fixes to another pr

Reply:
I think these formatting fixes are too few to deserve to propose another PR.

Reply:
a separate commit in this PR is also fine.

Reply:
It sounds good.

Reply:
all of these `test_get_container_id`  formatting changes can go into 5215746a32878641ce213086de3c2fce5af050bc

Reply:
Actually, this PR affects almost all files.  It is hard and unbeneficial to separate new codes and format fixing into two independent parts completely. Therefore the last commit contains a part of formatting changes, moving all of them seems not necessary.

Reply:
it's easy to pick the lines to you want using `git add -p ...`

Reply:
Sure.